lib/ansible/modules/system/known_hosts.py

```diff
@@ -323,20 +325,21 @@ def compute_diff(path, found_line, replace_or_add, state, key):
     diff['after'] = ''.join(lines)
     return diff
 
+
 def main():
 
     module = AnsibleModule(
         argument_spec = dict(
-            name      = dict(required=True,  type='str', aliases=['host']),
-            key       = dict(required=False,  type='str'),
+            name      = dict(required=True, type='str', aliases=['host']),
+            key       = dict(required=False, type='str'),
             path      = dict(default="~/.ssh/known_hosts", type='path'),
-            hash_host = dict(required=False, type='bool' ,default=False),
-            state     = dict(default='present', choices=['absent','present']),
-            ),
-        supports_check_mode = True
-        )
+            hash_host = dict(required=False, type='bool', default=False),
+            state     = dict(default='present', choices=['absent', 'present']),
```

Comment:
If you are fixing pep8 please remove all of these spaces so we can remove this file from legacy.

I see there was a comment here before saying someone liked the formatting better originally but you seem to have squashed commits removing history.

Unfortunately, if that is what the other comment was for, it was wrong.